### PR TITLE
WIP: Fixes #2205 – Refine margins and paddings around hero element

### LINKED ISSUE #2205: Hero image on non-desktop
screen shot 2018-03-06 at 2 27 35 pm

I was pretty sure this was reported already, but couldn't find it.

### PR DESCRIPTION
**Pull request by Regaddi**
r? @zoepage

### CODE REVIEW

File: webcompat/static/css/src/grid.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .grid {
-  margin: 9px;
+  margin: 9px 0;
```
Comment (zoepage) [inline webcompat/static/css/src/grid.css]: Should we use the var here too?

File: webcompat/static/css/src/variables.css [hunk 2]
```diff
@@ -1,4 +1,6 @@
 :root {
+  --grid-edge-s: 9px;
```
Comment (zoepage) [inline webcompat/static/css/src/variables.css]: As we deal with compatibility issues, edge might be misleading? 😅
How about we change it to `outermargin` or something similar?

File: webcompat/static/css/webcompat.dev.css [hunk 3]
```diff
@@ -40,18 +40,18 @@
 
 #body-webcompat {
   background-color: var(--color-bg-page);
-  margin-top: var(--navi-height);
+  padding: var(--navi-height) var(--grid-edge-s) 25px;
```
Comment (zoepage) [inline webcompat/static/css/webcompat.dev.css]: could we change the `25px` to `--unit-gutter-m` as this should work context wise?

File: webcompat/static/css/webcompat.dev.css [hunk 4]
```diff
@@ -40,18 +40,18 @@
...
 }
 
 @media screen and (min-width: 665px) {
   #body-webcompat {
-    padding: var(--navi-height) 19px 50px;
+    padding: var(--navi-height) var(--grid-edge-m) 50px;
```
Comment (zoepage) [inline webcompat/static/css/webcompat.dev.css]: I know, we used a fix value here, but could we change the `50px` as we are on it anyway to `calc(--unit-gutter-m * 2)`?

File: webcompat/static/css/src/grid.css (hunk 1 above)
Comment (Regaddi) [reply webcompat/static/css/src/grid.css]: This is probably not related / a side effect. It was the initial overall margin for all `.grid` elements and therefore produced the left and right margin.

Comment (zoepage) [reply webcompat/static/css/src/grid.css]: Just to make sure, we don't misunderstand each other, this was referring to the `9px` and if we should change this to `--grid-outermargin-s`. But this is 100% not blocking and we are free to merge!

**Comment by zoepage** [comment]
Merging this as the last comment is nothing pressuring. 

Great work @Regaddi && thank you so much! :)

**Pull request merged**